doc/contributing/releasing.md

```diff
@@ -31,10 +35,20 @@ We aim to eventually automate this process as much as possible.
 
 1. Go to the [NPM publish workflow](https://github.com/open-telemetry/opentelemetry-js/actions/workflows/publish-to-npm.yml)
 2. Click "Run workflow" (from main)
-   1. In rare cases not all packages are published due to a race when publishing, if you suspect this to
-      be the case, re-run the workflow: there should be enough time from 1.
+3. Get another maintainer to approve the workflow run
+   1. Have them navigate to the workflow run, and then click on "Review pending deployments" ![workflow job waiting for deployment approval](./releasing/waiting-for-approvals.png)
+   2. They should then check the box and select "Approve and deploy" ![approve and deploy button](./releasing/approve-and-deploy.png) to approve the deployment to NPM.
+
+## 4. Troubleshooting NPM publishing issues
+
+> [!NOTE]
+> You can skip this step if the `publish-to-npm` workflow completed successfully.
+
+- New packages that have never been published before cannot be published this way - contact an `@opentelemetry` org Admin to publish them manually.
```

Comment:
this is intentionally left vague - it does not happen too often and I'll add info/screenshots the next time I have to do it.